### DISCUSSION TITLE
Will plex_debrid keep the movie/tv files downloaded after watching or will it automatically delete the files

### DISCUSSION THREAD

**Discussion by NotoriousYeet** [itsToggle/plex_debrid#309] [Q&A]
As title states, I just want to know how much storage to set aside.

**Comment by itsToggle** [comment]
Okay it seems that I need to adjust the readme, as multiple people seem to miss this. The whole purpose of plex_debrid is to stream debrid content, rather than actually downloading it to your server.
Nothing is ever actually stored on your harddrive when using a plex_debrid setup, everything is streamed. 

The streaming part is accomplished by "rclone" or "rclone_rd", a program that can trick your OS into thinking you have your debrid service files locally (while in reality, they are streamed when you open them). This is why you will have unlimited storage (depending on your debrid service) and why content is available almost instantly. This is the first part of the setup process.

The second part of the setup process regards setting up a Plex server, so that you can watch your servers files from anywhere.

The last part of the setup is the plex_debrid script. It's used to easily add new content to your debrid services.

**Reply by mash2k3** [reply]
Which debrid service is unlimited? and which one do you prefer?

**Reply by itsToggle** [reply]
realdebrid. It's one of the cheapest and it has unlimited storage/traffic 👍

**Reply by NovaZazen** [reply]
I'm trying to set this up on my synology nas. Is there any special approach to be taken with a synology nas?
